packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -366,6 +374,16 @@ export class GitQuickOpenService {
         }
     }
 
+    private async getRemotesVerbose(): Promise<Remote[]> {
+        const repository = this.getRepository();
```

Comment:
[minor]: I would use one `getRemote` method in this class instead of `getRemotes` and `getRemotesVerbose`. It is completely OK to delete `private` fields and methods.

Reply:
I've removed the unnecessary `getRemotesVerbose` call, and refactored the existing `getRemotes`.